### DISCUSSION TITLE
Commit review error: " Oops, something went wrong" indefinitely

### DISCUSSION THREAD

**Discussion by VDiabloV** [community/community#78920] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi there, I'm having an issue with loading all the diffs for a commit I'm trying to view. Another developer reviewed and left comments, but whenever I try to view the commit it keeps prompting me to retry with no success. I'm assuming it's got to do with loading so much data but even on a 900mbs line, the diffs still fail to load. Is there another way to review commit comments on Git Hub to workaround this issue? 

What makes me think this is a bug is that when I click the "Ignore white spaces" button all diffs load fine, but then there is the problem of not displaying the comments anymore. Not sure why that's the case, I'd assume you'd still want to see comments when ignoring white space changes. 

![OopsSomethingWentWrong](https://github.com/community/community/assets/22020730/37ea60db-a620-413a-a4e3-b09b163a3ae0)

**Comment by marcindulak** [comment]
Previously reported at https://github.com/orgs/community/discussions/37991
